timely/src/dataflow/channels/pact.rs

```diff
@@ -68,7 +68,7 @@ where
 // Exchange uses a `Box<Pushable>` because it cannot know what type of pushable will return from the allocator.
 impl<T: Timestamp, C, H: 'static> ParallelizationContract<T, C> for ExchangeCore<C, H>
 where
-    C: ExchangeData + PushPartitioned + crate::dataflow::channels::ContainerBytes,
+    C: Data + Send + PushPartitioned + crate::dataflow::channels::ContainerBytes,
```

Comment:
Removes the requirement that containers implement `Serializable + for<'a> Deserializable<'a>`.